tests/components/nest/test_api.py

```diff
@@ -59,7 +59,6 @@ async def async_new_subscriber(
         """Capture credentials for tests."""
         nonlocal captured_creds
         captured_creds = creds
```

Comment:
Should we keep these lines as normal comment?

Reply:
I reverted the comment, thanks!